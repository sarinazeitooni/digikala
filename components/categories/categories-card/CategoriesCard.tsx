import React from 'react';
import style from './scss/categories-card.module.scss';
import ComputerOutlinedIcon from '@material-ui/icons/ComputerOutlined';
import BuildOutlinedIcon from '@material-ui/icons/BuildOutlined';
import ChildCareOutlinedIcon from '@material-ui/icons/ChildCareOutlined';
import ShoppingBasketIcon from '@material-ui/icons/ShoppingBasket';
import FavoriteOutlinedIcon from '@material-ui/icons/FavoriteOutlined';
import HomeOutlinedIcon from '@material-ui/icons/HomeOutlined';
import BorderColorOutlinedIcon from '@material-ui/icons/BorderColorOutlined';
import GrainOutlinedIcon from '@material-ui/icons/GrainOutlined';
import ChangeHistoryOutlinedIcon from '@material-ui/icons/ChangeHistoryOutlined';
interface props{
    icon : string,
    title : string,
    product : string
}
const CategoriesCard=({icon , title , product} : props)=>{
    return(
        <div className={style['categories-container']}>
            <div className={style['categories-icon']}><ComputerOutlinedIcon/></div>
            <div className={style['categories-title']}>{title}</div>
            <div className={style['categories-product']}> + {product}</div>
        </div>
    )
}
export default CategoriesCard;