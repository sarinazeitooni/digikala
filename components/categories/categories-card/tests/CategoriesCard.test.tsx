import {shallow} from "enzyme";
import CategoriesCard from "../CategoriesCard";
describe("Categories Card component" , ()=>{
    it("should render correctly" , ()=>{
        const container = shallow(<CategoriesCard/>);
        expect(container).toMatchSnapshot();
    })
});