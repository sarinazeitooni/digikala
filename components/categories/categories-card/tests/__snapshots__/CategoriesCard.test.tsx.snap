// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories Card component should render correctly 1`] = `ShallowWrapper {}`;
